### ISSUE TITLE
Samsung galaxy S23 Ultra with one ui 6

### ISSUE THREAD

**Issue by 2da2sa** [AndreyPavlenko/Fermata#505]
Same problem here. I have a samsung galaxy s23 Ultra with android 14 and one ui 6 and I only see fermata service. Neither fermata auto nor fermata mirror or anything at all appears. Using a wireless intermediate device does not work either. Once I made it work by downgrading from android 14 to android 13, installing everything there, that is, fermata auto, and once everything was installed there, update to android 14 and everything was working, but due to problems, I had to reset the phone and I am not able to find the official Samsung rom for Android 13 with binary 5... At the moment I cannot enjoy fermata auto :/ :(

**Comment by AndreyPavlenko** [comment]
The only wireless intermediate device that is known to work is AAWireless.

**Comment by ShelbyGT35** [comment]
> The only wireless intermediate device that is known to work is AAWireless.

Right, I'm using this adapter since several months and it was working perfect. Since appr.  2 weeks fermata isn't longer displayed in AA!!

**Comment by ShelbyGT35** [comment]
Tried several AA versions and with AA 11. 7.641474 ist's running. Not working under AA 12.3.142604!!

**Comment by AndreyPavlenko** [comment]
Check if you can see the app in the emulator - https://github.com/AndreyPavlenko/Fermata/issues/503#issuecomment-2183267654

**Comment by 2da2sa** [comment]
> Tried several AA versions and with AA 11. 7.641474 ist's running. Not working under AA 12.3.142604!! Insofar for me the problem is solved at the moment.

Not work in my s23!!!

**Comment by 2da2sa** [comment]
Why i download a fermata for playstore and i se nos fermata in AA? But no se youtube and browser. The rest of fermata github, donde show in AA one ui 6.1 Android 14

**Comment by mhmt05** [comment]
Hello,
I have the same problem. What can I do?

**Comment by YasmaniG88** [comment]
I don't know what's happening.   I bought the premium version of aaapk thinking it was the solution but no.  I hope that soon there will be a solution for the s23ultra

**Comment by YasmaniG88** [comment]
https://github.com/AndreyPavlenko/Fermata/assets/144571899/75eabf76-54fd-485b-9cac-78d658bfffc0

After reading all your comments I decided to try the developer's solution.  Finally it works!!!  AAWIRELES is the final solution. Thanks Andrey!!

**Comment by YasmaniG88** [comment]
I shouted victory, a few minutes later the image froze, it restarted and now it doesn't work again. :(

**Comment by dreimer1986** [comment]
> Why i download a fermata for playstore and i see now fermata in AA? But not see youtube and browser. The rest of fermata github, dont show in AA one ui 6.1 Android 14

Of course it does not show them, read the PlayStore version description and you will understand that this is crippled version as only no video variants are allowed in Play Store.

> I don't know what's happening. I bought the premium version of aaapk thinking it was the solution but no. I hope that soon there will be a solution for the s23ultra

Well, you already found out that AAAPK is useless on Android 14+ as it seems. I still have no clue why it hangs for you though. I use Youtube playing Music videos all the time in my car. Pixel 7 Pro Android 15 AAWireless in Debug Mode.

**Comment by 2da2sa** [comment]
I had to insert an official Samsung rom, because root causes me to lose automatic actions, NFC payment with bank accounts, linking with the SmartWatch and a lot of other problems. So now that I'm back without root, I'm having the same problems as before again. I use AA with an AAWireless from China, I don't know if I can do one of the solutions for a device without root...
The device is this:
I just found this amazing item on AliExpress. Check it out! 
€57.05 | Birgus C1-AA Android Car Wireless Device for Toyota Mazda Nissan Camry Suzuki Subaru Citroen Audi Mercedes Kia Ford
https://a.aliexpress.com/_EIcVm8R

Any solution with this? Thank you

**Comment by 2da2sa** [comment]
AndreyPavlenko do magic!!!

**Comment by JVKeller** [comment]
> I had to insert an official Samsung rom, because root causes me to lose automatic actions, NFC payment with bank accounts, linking with the SmartWatch and a lot of other problems. So now that I'm back without root, I'm having the same problems as before again. I use AA with an AAWireless from China, I don't know if I can do one of the solutions for a device without root... The device is this: I just found this amazing item on AliExpress. Check it out! €57.05 | Birgus C1-AA Android Car Wireless Device for Toyota Mazda Nissan Camry Suzuki Subaru Citroen Audi Mercedes Kia Ford https://a.aliexpress.com/_EIcVm8R
> 
> Any solution with this? Thank you

The official AAWireless dongle is on sale for PrimeDay $56.24 (USD)

**Comment by dreimer1986** [comment]
The official one works fine here and it's a older v1 one from the frundraiser campaign. No clue what the clone does, but I think your problem might be that clone variant.

**Comment by onoff7** [comment]
> C1-AA

Is this copy of AAWireless really capable of using Fermata on Android 14?
Are other known copies currently available on Aliexpress known?
I'm just from Russia, and AAWireless is not available for purchase here.

**Comment by JVKeller** [comment]
AAWireless works.

No one has really reported any of the clones working.

The trick seems to be the companion app that enables dev mode on the
dongle. If it has that capability, then maybe, if not, I wouldn't bother
trying.

**Comment by tungdt-90** [comment]
S21 ultra with one ui 6.1 here. I use wireless AA dongle as well, but the Fermata does not show up in AA. I've installed with King installer. Any one can help?
